superduperdb/container/component.py

```diff
@@ -45,6 +47,14 @@ def on_load(self, db: DB) -> None:
     def child_components(self) -> t.Sequence[t.Any]:
         return []
 
+    @cached_property
+    def serialized(self):
```

Comment:
This can create side effects.

Reply:
For example, the version assigned by `db.add`.

Reply:
done